src/changes/changes.xml

```diff
@@ -156,7 +156,7 @@
         Remove deprecated code.
       </action>
       <action issue="LOG4J2-2545" dev="ckozak" type="fix">
-        RoutingAppender.BuilderlsetPurgePolicy fluently returns the builder instance.
+        RoutingAppender.Builder.setPurgePolicy fluently returns the builder instance.
```

Comment:
Misleading changelog entries.

Although some projects prefer not to change historical entries in their changelogs, in which case it's trivial for me to drop all changes to this file.